pkg/compiler/native_test.go

```diff
@@ -0,0 +1,28 @@
+package compiler
+
+import (
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/designate"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/gas"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/neo"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/oracle"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/policy"
+	"github.com/stretchr/testify/require"
+)
+
+func TestContractHashes(t *testing.T) {
+	cs := native.NewContracts(false)
+	require.Equal(t, []byte(neo.Hash), cs.NEO.Hash.BytesBE())
```

Comment:
This allows to remove compatibility test from `native`.

Reply:
I think it is a bit different, but fixed.